modules/microshift-get-mirror-reg-container-image-list.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-get-mirror-reg-container-image-list_{context}"]
+= Getting the {product-title} mirror registry container image list
+
+To use a mirror registry, you must know which container image references are used by a specific version of {product-title}. These references are provided in the `release-<arch>.json` files that are part of the `microshift-release-info` RPM package.
+
+.Procedure
+
+. Access the list of container image references by using one of the following methods:
+
+.. If the package is installed on the {product-title} host, get the location of the files by running the following command:
++
+[source,terminal]
+----
+$ rpm -ql microshift-release-info
+/usr/share/microshift/release/release-aarch64.json
+/usr/share/microshift/release/release-x86_64.json
+----
+
+.. If the package is not installed on a {product-title} host, download and unpack the RPM package without installing it by running the following command:
++
+[source,terminal]
+----
+$ rpm2cpio microshift-release-info*.noarch.rpm | cpio -idmv
+./usr/share/microshift/release/release-aarch64.json
+./usr/share/microshift/release/release-x86_64.json
+----
+
+. Extract the list of container images into the `microshift-container-refs.txt` file by running the following command:
++
+[source,terminal]
+----
+$ RELEASE_FILE=/usr/share/microshift/release/release-$(uname -m).json
+$ jq -r '.images | .[]' ${RELEASE_FILE} > microshift-container-refs.txt
```

Comment:
See note in [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc) about avoiding `jq`, if possible. Otherwise, might need to list it as a prerequisite, at the very least.